activerecord/CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
 
 ## Rails 5.0.0 (June 30, 2016) ##
 
+*   AM::Dirty's changed? methods now return an actual boolean, never `nil`, as in 4.2.
```

Comment:
We should spell out `ActiveModel::Dirty`, wrap `changed?` in backticks and move this to the correct changelog 😁

We also prefer `singleton` to `boolean`. "actual" reads a bit condescending to me, I'd snip that.

You say "methods" but `changed?` is only one, were you referring to the `*_changed?` methods?